apps/experiments/tests/test_views.py

```diff
@@ -252,3 +253,39 @@ def test_new_participant_created_for_different_teams(_trigger_mock, is_user):
     # There should be two participants with external_chat_id = identifier accross all teams
     assert Participant.objects.filter(external_chat_id=identifier).count() == 2
     assert session.participant.external_chat_id == identifier
+
+
+@pytest.mark.django_db()
+@mock.patch("apps.experiments.views.experiment.enqueue_static_triggers")
+def test_participant_gets_user_when_they_signed_up(_trigger_mock, client):
```

Comment:
I made an oopsy with my commits. [This test](https://github.com/dimagi/open-chat-studio/pull/363/commits/2a26b181931560770905221b5161b8639cae427c) should have been included in this commit. This test you see here should be in [this commit](https://github.com/dimagi/open-chat-studio/pull/363/commits/23e5611f97ee507b613fecbd7a3f5c5258b65b62)